ssz/src/main/java/org/ethereum/beacon/ssz/access/SSZField.java

```diff
@@ -25,6 +26,17 @@ public static SSZField resolveFromValue(Object value, Class<?> clazz) {
       return new SSZField(
           new ParametrizedTypeImpl(clazz, ((List) value).get(0).getClass()));
     }
+    if (value instanceof ReadList && !((ReadList) value).isVector() && ((ReadList) value).maxSize() > VARIABLE_SIZE) {
+      ReadList readList = (ReadList) value;
+      Class elementClass = UInt64.class; //XXX: doesn't matter
```

Comment:
If it matters nothing then why not to use an `Object.class`?

Reply:
It should be any number in the end